root/static/scripts/edit/URLCleanup.js

```diff
@@ -536,7 +541,7 @@ const CLEANUPS = {
       return url;
     },
     validate: function (url, id) {
-      const m = /^https:\/\/www.bandsintown\.com\/(?:(a|e|v)\/)?([^\/?#]+)$/.exec(url);
+      const m = /^https:\/\/www.bandsintown\.com\/(?:[aev]\/)?([^\/?#]+)$/.exec(url);
```

Comment:
This looks like it might break something too.

Reply:
Because this removes a capturing group, thus modifies match results `m[X]`, see https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp#grouping-back-references

Reply:
This is most probably what caused test failure at https://circleci.com/gh/metabrainz/musicbrainz-server/87. Note: you can run these tests locally with the command:
```shell
node root/static/scripts/tests/Control/URLCleanup.js
```

Reply:
Oooh, that's what m[x] meant. Sorry, thought it would be match 1 or something :) ok, will check.